### ISSUE TITLE
Tell us the Technologies (not languages) your project using

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/website#459]
### Overview
Volunteers want to know what frameworks projects use, so that they can better choose which project to join.

### Action Items
Write a comment in this issue with:
1. the name of your project
2. the frameworks you use (e.g., React-JS, Vue, Angular, Ember, Backbone, Express, Django, Rails, etc.)

**Comment by ddfridley** [comment]
1. Undebate
2. Mongo, Express, React, Node (MERN)

**Comment by theswerd** [comment]
1. New Schools Today
2. ReactJS, Flutter

**Comment by eselkin** [comment]
1. Engage
2. React, Django, SQL (PostgreSQL)

**Comment by c-fuchs** [comment]
1. HomeUnite.Us
2. React, NoSQL

**Comment by ExperimentsInHonesty** [comment]
I am going to message each of the projects directly throughout the day.

311-data slack message: https://hackforla.slack.com/archives/CJY4R382X/p1587920983101900
FOLA slack message: https://hackforla.slack.com/archives/CMER3R1RD/p1587922553171400
HelloGov slack message: https://hackforla.slack.com/archives/CDA23KHKP/p1587922621006600
Metro-Ontime sent cam a private message since project is completed.
Write for All slack message: https://hackforla.slack.com/archives/CTJN4HPRA/p1587925379142200
TDM calculator slack message: https://hackforla.slack.com/archives/CKY65G266/p1587925315140100
Lucky Parking slack message: https://hackforla.slack.com/archives/CPFDMSV6V/p1587915685024500
Public Tree map slack message: https://hackforla.slack.com/archives/C9UU5KKAA/p1587925293029600
VRMS slack message: https://hackforla.slack.com/archives/CRGH5HM0Q/p1587925343005000

**Comment by ExperimentsInHonesty** [comment]
1. Hack for LA website
2. gh-pages, jekyll

@cnk question for you.  Why does it not show liquid as a language?

**Comment by breeze-094** [comment]
1. Lucky Parking
2. React-JS, PostgreSQL

**Comment by cnk** [comment]
> @cnk question for you. Why does it not show liquid as a language?

Because we do not use a .liquid file extension for our template files.

**Comment by johnr54321** [comment]
For 311:
Javascript: React / Redux
Python: Flask (Sanic) / Pandas
CSS: Bulma
DB: PostgreSql

**Comment by wesrowe** [comment]
@harishlingam @yuikomajima or potentially others — I was just looking for the best entry point (link) to give to reactJS developers. 
I'll say at the outset, maybe I'm missing something, or maybe you already have this in the planning stage... 
**Use case:** I'm trying to market onboarding nights on social media. Most design/tech communities are organized around specific flavors of tech or design -- so a social media message (eg tweet) is going to be tailored to, e.g., ux designers, react devs, javascript devs, even devops and QA – by using hashtags. I just this afternoon tweeted at (hashtags for) nodeJS and reactJS.
Linking for a link to add to the tweet, I found the info a little fragmented on the current website. I wrote a tweet-thread to give people tips on finding their way around, but the experience of finding e.g. a react project is clunky:
- homepage has "current needs" in leads' words (arguably a little hard to parse sometimes). Can be helpful if someone takes the time to read carefully.
- project cards have languages (and in the future, frameworks, right?).
_Do you have a suggestion for the best place to land a skill-focused visitor? or a feature idea you could prioritize?_

**Comment by ExperimentsInHonesty** [comment]
@ExperimentsInHonesty to do another round of encouraging reporting

**Comment by ExperimentsInHonesty** [comment]
Here are the projects that don't have the Technologies listed
civic tech index
hellogov
TDM
Not Today
Record Clearance
Public Tree Map
VRMS
Write for All
Food Oasis
Heart
Railstats
Work for LA
Jobs for Hope
Shared Housing 
Spare

**Comment by ExperimentsInHonesty** [comment]
Please see the image below.  The one with the arrow is a project we have the technologies listed. 
 The project without, doesn't have it.  We need your Project's Technologies for the new project filtering that is getting added to the website.  ACTION ITEM: ADD COMMENT TO ISSUE WITH YOUR UPDATE
![Screen Shot 2020-05-24 at 9 52 14 AM](https://user-images.githubusercontent.com/37763229/82759784-479eac80-9da4-11ea-85fa-d25aedfcd59f.png)

**Comment by ExperimentsInHonesty** [comment]
Next task is to make a spreadsheet of who has technologies listed and who doesn't and then reach out to them with the following post

Hi, I am working on updating the website to make it easier for new volunteers to find projects.  We have this new area on your project cards called Technologies.  This includes things like React/Redux, Pandas, PostgresSQL, Flask, etc. that won't show up on the Languages item.  Can you look at the example provided here and post your update on the issue https://github.com/hackforla/website/issues/459#issuecomment-633259590

**Comment by jbubar** [comment]
here is the [google spreadsheet](https://docs.google.com/spreadsheets/d/12Yz_q2j_1eOa7Z3ekjqTPENFHs5sbjT98qMIApZ45ts/edit?usp=sharing) to keep track of each projects technologies.


Progress: I reached out to all of the projects (except for 100automations) that have not given us the technologies that they use.
Blocks: could not find a slack channel for 100automations
ETA: depending on how quickly the teams get back to me, but hopefully before Monday.
Availability: Most of the time over the weekend

**Comment by alex-anakin** [comment]
for VRMS: 
back-end: Node.js/Express
DB: MongoDB
front-end: React.js
styles: SCSS

**Comment by ExperimentsInHonesty** [comment]
Civic Opportunity:
None at this time.  We will fill this out when we get farther along.

**Comment by ExperimentsInHonesty** [comment]
100 automations: 
GitHub Pages, Jekyll, Python, Javascript

**Comment by ExperimentsInHonesty** [comment]
Racial Justice:
Researching on Google, email

**Comment by entrotech** [comment]
1. TDM
2. React/HTML/CSS,  Node/Express, Microsoft SQL Server, Docker, Azure App Servcie for Containers

**Comment by entrotech** [comment]
1. Food Oasis
2. Reach/HTML/CSS Mapbox, Material-UI, Node/Express, Postgres with some Postgis (geographic) features, AWS, docker

**Comment by cayd** [comment]
1. Record Clearance
2. React

**Comment by emillipede** [comment]
public tree map

- geopandas
- jquery
- leaflet
- node

**Comment by nickgieschen** [comment]
NotToday

react-native

**Comment by KianBadie** [comment]
Sorry, I wrongly added the "Fixes" keyword to a pull request for this. Reopening

**Comment by jbubar** [comment]
I messaged Kate from hellogov and Joel from write for all. Awaiting on their response. Updated public tree map and not today, waiting on a code review.

**Comment by jbubar** [comment]
Reached out to Joel from write for all again.  Have not heard back from hellogov.

**Comment by alexandrastubbs** [comment]
Closing this issue and proceeding without Hellogov. Heard from Joel at Write for All that they do not wish to provide updates any longer.
